messybrainz/cluster/cluster.py

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env python3
+
+import pika
+import sys
+import ujson
+from brainzutils import musicbrainz_db
+from brainzutils.musicbrainz_db.exceptions import NoDataFoundException
+from flask import current_app
+from messybrainz.cluster import utils
+from messybrainz import db
+from messybrainz.db import data
+from messybrainz.db import artist as db_artist
+from messybrainz.db import recording as db_recording
+from messybrainz.db import release as db_release
+from messybrainz.webserver import create_app
+from sqlalchemy.exc import IntegrityError
+from uuid import UUID
+
+
+class Cluster:
```

Comment:
this name should be `ClusterWriter` or something similar.

Reply:
Is RecordingAssociator a good name? Cz we are associating MSID to MBIDs.